vassal-app/src/main/java/VASSAL/counters/MatCargo.java

```diff
@@ -390,6 +424,8 @@ public HelpFile getHelpFile() {
     private final StringConfigurer descInput;
     private final BooleanConfigurer rotInput;
     private final TraitConfigPanel controls;
+    private final IntConfigurer XInput;
+    private final IntConfigurer YInput;
```

Comment:
Generally we don't capitalize private members.

Reply:
fixed